### ISSUE TITLE
Spatial Example: Weather Interpolation

### ISSUE THREAD

**Issue by cryptarch** [HarrietInc/elegant-scipy-submissions#8]
Code by @nfaggian
Submitted by @cryptarch

https://github.com/nfaggian/pyconau/tree/master/2014/notebooks

> See presentation and code ... including `sampler.py`

**Comment by jni** [comment]
@cryptarch sorry for the silence! Actually I really like this example, we just ran out of time. I'd like to keep it open as an idea for Edition 2! I hope that's ok! =)

**Comment by cryptarch** [comment]
Yeah, sure. Sorry, I wasn't trying to be passive aggressive :) Just not using GitHub as much these days, and I couldn't remember the context for this, so it seemed safe to close it. Up to you where it goes from here.
